src/components/Hero/Hero.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import Table from '../../assets/images/table.png';
+import {
+  HeroWrapper,
+  Title,
+  Subtitle,
+  HeroForm,
+  CTA,
+  Input,
+} from './Hero.style';
+
+function Hero({ title, subtitle, cta }) {
```

Comment:
I don't think you need props for this. I would just hard-code the strings inside the component since the Hero section is not reused yet